### PR TITLE
[RFR] Widget branding

### PR DESCRIPTION
**Pull request by eschaefer**
- [x] Sitename is reflected
- [x] Theme colors are reflected
- [x] Sitename is linked to home page

screen shot 2016-10-26 at 15 10 16

### CODE REVIEW

**Comment by eschaefer** [comment]
@artibella @mattberridge RFR

File: src/components/PetitionWidget/index.js [hunk 1]
```diff
@@ -2,21 +2,17 @@ import React from 'react';
...
 import WidgetSupportButton from 'containers/widget/WidgetSupportButton';
 import PetitionWidgetHeader from 'containers/widget/PetitionWidgetHeader';
-import PetitionFooter from 'containers/PetitionFooter';
 
 const PetitionWidget = () => (
   <article className={styles.root}>
     <Section>
       <Container>
         <PetitionWidgetHeader />
         <WidgetSupportButton />
```
Comment (mattberridge) [inline src/components/PetitionWidget/index.js]: re: naming, could it be `<PetitionWidgetSupportButton>` as it's petition-specific like the include above?

File: src/components/WidgetBranding/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <div className={styles.root}>
+    <span className={styles.text}>{settings.widgetLink.text} </span>
+    <span className={styles.link}>
+      <Link
+        href={baseUrl()}
+        text={settings.logo}
+        external
+        newTab
+      />
+    </span>
```
Comment (mattberridge) [inline src/components/WidgetBranding/index.js]: no semantic html elements?

File: src/components/WidgetSupportButton/index.js [hunk 3]
```diff
@@ -3,21 +3,18 @@ import ButtonLink from 'components/ButtonLink';
...
-        block
-        external
-        newTab
-        modifier={'accent'}
-      />
-    </TextCenter>
-  </div>
+  <TextCenter>
+    <ButtonLink
+      href={`${baseUrl()}/petitions/${petition.id}`}
```
Comment (mattberridge) [inline src/components/WidgetSupportButton/index.js]: we've a helper here :)
`getPetitionURL(petition.id)`

**Pull request merged**